app/_data/docs_nav_kuma_2.0.x.yml

```diff
@@ -19,6 +19,7 @@ items:
             url: "/introduction/how-kuma-works/#kuma-vs-xyz"
       - text: Deployments
         url: /introduction/deployments/
+        src: /production/deployment/
```

Comment:
I thought single source was only used from 2.1.x no? If so it seems like these page moves shouldn't happen in 2.0.x no?